testing/py3-blis/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=py3-blis
+pkgver=0.4.0
+pkgrel=0
+pkgdesc="Fast matrix-multiplication as a self-contained Python library"
+url="https://github.com/explosion/cython-blis"
+arch="all"
+license="MIT"
+depends="python3"
+makedepends="python3-dev py3-setuptools cython3"
+#checkdepends="py3-pytest py3-hypothesis py3-numpy py3-tox"
+checkdepends="py3-pytest py3-hypothesis py3-numpy"
+subpackages="$pkgname-doc"
+source="blis-$pkgver.tar.gz::https://github.com/explosion/cython-blis/archive/v$pkgver.tar.gz
+tox.ini"
+builddir="$srcdir/cython-blis-$pkgver"
+
+case "$CARCH" in
+	aarch64)
+		export BLIS_ARCH=cortex53
+		;;
+esac
+
+build() {
+	python3 setup.py build
+}
+
+check() {
+	python3 setup.py install --root="$PWD/tmp_install"
+
+	cd tmp_install/usr/lib/python3.7/site-packages
+
+	PYTHONPATH="$PWD/tmp_install/usr/lib/python3.7/site-packages:$PYTHONPATH" \
```

Comment:
overcomplicated.

PYTHONPATH="$PWD/build/lib" should be enough

Reply:
Done! Thank you!